localstack/config.py

```diff
@@ -655,6 +655,8 @@ def populate_edge_configuration(
     GATEWAY_LISTEN,
 ) = populate_edge_configuration(os.environ)
 
+GATEWAY_WORKER_COUNT = int(os.environ.get("GATEWAY_WORKER_COUNT") or 1000)
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_refinement):** Ensure environment variable fallback values are consistent across the codebase.

The fallback of 1000 for `GATEWAY_WORKER_COUNT` is used directly in multiple places. Consider defining this as a constant to ensure consistency and ease of maintenance.

```suggestion
        threads: int = os.cpu_count() or 4,
```